__tests__/app.test.js

```diff
@@ -0,0 +1,46 @@
+const request = require('supertest');
+const app = require('../app.js');
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed.js')
+const testData = require('../db/data/test-data');
+
+beforeEach(() => {
+    return seed(testData);
+});
+
+afterAll(() => {
+    return db.end();
+});
+
+describe('/api/topics', () => {
+    describe('GET', () => {
+        it('returns an array of topics that contain a description and a slug property', () => {
+            return request(app)
+                .get('/api/topics')
+                .expect(200)
+                .then(({ body: { topics } }) => {
+                    expect(topics).toBeInstanceOf(Array);
```

Comment:
You don't need to test if this is an instance of an array, it is implicitly done with your test, as you're checking length and looping through